### PR TITLE
*: add comment for show table status.

### PR DESCRIPTION
**Pull request by qiuyesuifeng**
Fix https://github.com/pingcap/tidb/issues/2012

### CODE REVIEW

**Comment by zimulala** [comment]
Fix CI @qiuyesuifeng

**Comment by hanfei1991** [comment]
LGTM

File: table/table.go [hunk 1]
```diff
@@ -143,6 +145,17 @@ const (
 	codeNoDefaultValue  = 1364
 )
 
+// Slice is used for table sorting.
+type Slice []Table
+
+func (s Slice) Len() int { return len(s) }
+
+func (s Slice) Less(i, j int) bool {
+	return strings.Compare(s[i].Meta().Name.L, s[j].Meta().Name.L) < 0
```
Comment (coocood) [inline table/table.go]: `s[i].Meta().Name.L < s[j].Meta().Name.L`

**Comment by qiuyesuifeng** [comment]
@coocood  PTAL

**Comment by coocood** [comment]
LGTM

**Pull request merged**